ch4/switch-enums.zig

```diff
@@ -0,0 +1,33 @@
+const std = @import("std");
```

Comment:
Is this file named correctly? In __Chapter 4__ the Enum was not introduced yet, also, there's no enum in this example

I know that enum comes from enumeration which is a list of items but for the target audience (programmers) it can be confusing